### PR TITLE
Add snippet for Android sound classification

### PR DESCRIPTION
**Pull request by khanhlvg**

### CODE REVIEW

File: snippets/markdown/audio/tflite/Android.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+You can test your TensorFlow Lite sound classification model on Android by following these steps:
+1. Download the [sample app](https://github.com/tensorflow/examples/tree/master/lite/examples/sound_classification/android) from GitHub.
+2. Extract the ZIP archive that you download from Teachable Machine.
+3. Copy the `soundclassifier.tflite` and `labels.txt` files from the archive to the `src/main/assets` folder in the sample app, replacing the demo model there.
+
+If you want to integrate the model into your existing app, follow these steps:
+1. Put the `soundclassifier.tflite` and `labels.txt` files into the `assets` folder in your app.
+2. Copy the [SoundClassifier.kt](https://github.com/tensorflow/examples/blob/master/lite/examples/sound_classification/android/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/SoundClassifier.kt) file to your app. This file contains the source code to use the sound classification model.
+3. Initialize a `SoundClassifier` instance from your `Activity` or `Fragment` class.
```
Comment (caisq) [inline snippets/markdown/audio/tflite/Android.md:9]: It seems to me that the code such as `soundClassifier.start()` and `soundClassifier.probabilities.observe()` is already included in the sample app. So the text here has the slight risk of making the readers think that the sample app is incomplete and they need to manually add those lines of code.

So perhaps make it clear that the code already exists, and the description here is just calling them out for highlighting?

**Review by caisq — COMMENTED** [review]
Thanks. See my nit comment below.

File: snippets/markdown/audio/tflite/Android.md (hunk 1 above)
Comment (khanhlvg) [reply snippets/markdown/audio/tflite/Android.md:11]: These steps are for integrating the SoundClassifier into an arbitrary app, which is different to the sample app. I added a heading to make it clearer. WDYT?

Comment (caisq) [reply snippets/markdown/audio/tflite/Android.md:11]: Looks good. I think it's clearer.

File: snippets/markdown/audio/tflite/Android.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Integrate your model into your own app
+If you want to integrate the model into your existing app, follow these steps:
+1. Put the `soundclassifier.tflite` and `labels.txt` files into the `assets` folder in your app.
+2. Copy the [SoundClassifier.kt](https://github.com/tensorflow/examples/blob/master/lite/examples/sound_classification/android/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/SoundClassifier.kt) file to your app. This file contains the source code to use the sound classification model.
+3. Initialize a `SoundClassifier` instance from your `Activity` or `Fragment` class.
+```kotlin
+var soundClassifier: SoundClassifier
+soundClassifier = SoundClassifier(context).also {
+    it.lifecycleOwner = context
```
Comment (Sa-ryong) [inline snippets/markdown/audio/tflite/Android.md:15]: It's more common to assign `this` in Activity, `viewLifecycleOwner` in Fragment.
How abt below?
```
  it.lifecycleOwner = this // this should be `viewLifecycleOwner` when called in Fragment.
```

File: snippets/markdown/audio/tflite/Android.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+```
+4. Start capturing live audio from the device's microphone and classify in realtime:
+```
+soundClassifier.start()
+```
+5. Receive classification result in realtime as a map of human-readable class name and probabilities of the current sound belonging to each particular category. 
+```kotlin
+let labelName = soundClassifier.labelList[0] // e.g. "Clap"
+soundClassifier.probabilities.observe(this) { resultMap ->
+    let probability = result[labelName] // e.g. 0.7
```
Comment (Sa-ryong) [inline snippets/markdown/audio/tflite/Android.md:26]: typo: need to be `resultMap[lableName]`

Comment (khanhlvg) [reply snippets/markdown/audio/tflite/Android.md]: done

File: snippets/markdown/audio/tflite/Android.md (hunk 2 above)
Comment (khanhlvg) [reply snippets/markdown/audio/tflite/Android.md]: done

**Comment by HalfdanJ** [comment]
If you run `npm run lint` theres a couple of simple lint errors in the markdown file. 

Here's what currently triggering error:

```
Step #1: markdown/audio/tflite/Android.md: 1: MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "## Test your model with our sample app"]
Step #1: markdown/audio/tflite/Android.md: 3: MD013/line-length Line length [Expected: 100; Actual: 136]
Step #1: markdown/audio/tflite/Android.md: 3: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "1. Download the [sample app](h..."]
Step #1: markdown/audio/tflite/Android.md: 5: MD013/line-length Line length [Expected: 100; Actual: 159]
Step #1: markdown/audio/tflite/Android.md: 7: MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "## Integrate your model into your own app"]
Step #1: markdown/audio/tflite/Android.md: 9: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "1. Put the `soundclassifier.tf..."]
Step #1: markdown/audio/tflite/Android.md: 10: MD013/line-length Line length [Expected: 100; Actual: 301]
Step #1: markdown/audio/tflite/Android.md: 11: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "3. Initialize a `SoundClassifi..."]
Step #1: markdown/audio/tflite/Android.md: 12: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```kotlin"]
Step #1: markdown/audio/tflite/Android.md: 18: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
Step #1: markdown/audio/tflite/Android.md: 19: MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 4; Style: 1/1/1]
Step #1: markdown/audio/tflite/Android.md: 19: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "4. Start capturing live audio ..."]
Step #1: markdown/audio/tflite/Android.md: 20: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
Step #1: markdown/audio/tflite/Android.md: 20: MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]
Step #1: markdown/audio/tflite/Android.md: 22: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]
Step #1: markdown/audio/tflite/Android.md: 23: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
Step #1: markdown/audio/tflite/Android.md: 23: MD013/line-length Line length [Expected: 100; Actual: 161]
Step #1: markdown/audio/tflite/Android.md: 23: MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 5; Style: 1/1/1]
Step #1: markdown/audio/tflite/Android.md: 23: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "5. Receive classification resu..."]
Step #1: markdown/audio/tflite/Android.md: 24: MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```kotlin"]
```

**Comment by khanhlvg** [comment]
I've fixed the lint errors.

**Comment by GautamBose** [comment]
LGTM! thanks for adding the notes on the emulator!

**Pull request merged**